### PR TITLE
check-process.rb: return CRITICAL if pid file is specified and does not exist

### PR DESCRIPTION
**Pull request by swibowo**
## Pull Request Checklist

Addressing #23 . 

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose

I encountered the same issue.

If pid file is specified, it is expected that the pid file must exist and contain pid to match. If pid file becomes non-existent, it very likely means that the monitored process cease to run (or the check should be treated as 'fail' because the pid file given is invalid) and check-process.rb check is expected to return CRITICAL rather than UNKNOWN. 

#### Known Compatablity Issues

Technically, none.

### CODE REVIEW

**Comment by majormoses** [comment]
@swibowo mentioned just now on the issue I think that should be configurable option to fail on pid not found. I would say it should default to false to maintain backwards compatibility.

**Comment by swibowo** [comment]
You raised a good point there @majormoses. I've modified the pull request to reflect the said changes.

**Comment by majormoses** [comment]
@swibowo any reason to not make it an actual argument and then we can still use unknown for users that want to keep existing behavior?

**Comment by swibowo** [comment]
@majormoses , I've reviewed the logic and per your suggestion, added a new option `-F` to trigger a critical if pid file is non-existent. If `-F` is not specified, an unknown is triggered (backwards compatible).

File: bin/check-process.rb [hunk 1]
```diff
@@ -176,8 +181,10 @@ class CheckProcess < Sensu::Plugin::Check::CLI
   def read_pid(path)
     if File.exist?(path)
       File.read(path).strip.to_i
-    else
+    elsif config[:file_pid_crit].nil?
```
Comment (majormoses) [inline bin/check-process.rb]: shouldn't this be `!config[:file_pid_crit].nil?` as you are wanting to crit when its specified which without I believe would be nil.

**Comment by majormoses** [comment]
@swibowo now I'm a maintainer so I can merge this now...

**Pull request merged**